packages/backend/src/paymentPointerKey/model.ts

```diff
@@ -2,14 +2,14 @@ import { BaseModel } from '../shared/baseModel'
 
 import { JWKWithRequired } from 'auth'
 
-export class ClientKeys extends BaseModel {
+export class PaymentPointerKey extends BaseModel {
```

Comment:
I'm tempted to try to make this work:
```suggestion
export class PaymentPointerKey extends PaymentPointerSubresource {
```
which would let you utilize the existing [`list` query](https://github.com/interledger/rafiki/blob/92c89a35aca2cfde7903abbef04fcfffb2b6f369/packages/backend/src/open_payments/payment_pointer/model.ts#L141).
See:
https://github.com/interledger/rafiki/blob/92c89a35aca2cfde7903abbef04fcfffb2b6f369/packages/backend/src/open_payments/payment/outgoing/service.ts#L357-L362
Note that:
https://github.com/interledger/rafiki/blob/92c89a35aca2cfde7903abbef04fcfffb2b6f369/packages/backend/src/open_payments/payment/outgoing/service.ts#L32-L33
At the moment, `PaymentPointerSubresource` (added in https://github.com/interledger/rafiki/pull/647) requires an `assetId` though...

Reply:
@adrianhopebailie and I talked about this and if we assume not every `PaymentPointerSubresource` should have an `assetId`, it would require a change to `PaymentPointerSubresource` in order to `PaymentPointerKey` implement it. I think we can create a new PR and make the `assetId` an explicit property to `Quote`, `IncomingPayment`, and `OutgoingPayment` models there.

Reply:
- https://github.com/interledger/rafiki/issues/701